### DISCUSSION TITLE
How to Install on Windows?

### DISCUSSION THREAD

**Discussion by releyshic** [gildas-lormeau/single-file-companion#3] [Q&A]
This commands "Install [Node.js](https://nodejs.org/)
Install [jq](https://stedolan.github.io/jq/), the JSON processor
Download the [SingleFile Companion project](https://github.com/gildas-lormeau/single-file-companion/archive/master.zip) zip file and unzip it somewhere on your disk.
Run npm install in the root folder where you unzipped SingleFile Companion.
Go into the subfolder corresponding to your OS and your browser. " worked Ok
But what i most to do next? 
Instruction said "Run install
install.bat (Windows)" How to make it?

**Comment by gildas-lormeau** [comment]
Run `install.bat` on the command line or the explorer. Nothing particular will happen but if it works properly, you should be able to use the Companion as `Destination` in the options page of SingleFile.
